agent/battle/perception/perception.py

```
@@ -205,34 +206,34 @@ def _detect_servants(context, img) -> Tuple[ServantState, ...]:
         cd_result = _reco(context, cd_node, img)
         if cd_result and cd_result.hit:
-            # 命中 CD "剩余" 文字 → 技能在冷却，不可用。
+            # 命中 CD 数字模板 → 技能在冷却，不可用。
             score = getattr(getattr(cd_result, "best_result", None), "score", 1.0) or 1.0
-            out.append(SkillState(False, Confidence(float(score), "ocr:cd")))
+            out.append(SkillState(False, Confidence(float(score), "template:cd")))
             continue
-        # 未命中任何 CD 文字 → 技能可用。
-        out.append(SkillState(True, Confidence(1.0, "ocr:no_cd")))
+        # 未命中任何 CD 数字 → 技能可用。
+        out.append(SkillState(True, Confidence(1.0, "template:no_cd")))
```

Ruff (0.16.3)

[warning] Boolean positional value in function call

(FBT003)